runtime/domain.c

```diff
@@ -335,36 +354,36 @@ void caml_domain_set_name(char *name)
 
 asize_t caml_norm_minor_heap_size (intnat wsize)
 {
-  asize_t bs, max;
+  asize_t bs;
   if (wsize < Minor_heap_min) wsize = Minor_heap_min;
   bs = caml_mem_round_up_pages(Bsize_wsize (wsize));
 
-  max = Bsize_wsize(Minor_heap_max);
-
-  if (bs > max) bs = max;
-
   return Wsize_bsize(bs);
 }
 
 /* The current minor heap layout is as follows:
 
-- A contiguous memory block of size [Minor_heap_max * Max_domains]
-  is reserved by [caml_init_domains]. The boundaries of this
-  reserved area are stored in the globals
+- A contiguous memory block of size
+   [caml_minor_heap_max_wsz * Max_domains]
+  is reserved by [caml_init_domains]. The boundaries
+  of this reserved area are stored in the globals
     [caml_minor_heaps_base]
   and
-    [caml_minor_heaps_base_end}.
+    [caml_minor_heaps_base_end].
 
-- Each domain gets a reserved section of this block of size
-  [Minor_heap_max], whose boundaries are stored as
-    [domain_self->minor_heap_base]
+- Each domain gets a reserved section of this block
+  of size [caml_minor_heap_max_wsz], whose boundaries are stored as
+    [domain_self->minor_heap_area]
```

Comment:
Generally, in this section, the terminology of the pointer that points to the end of an area is consistent but not the one that points to the start. See

1. `caml_minor_heaps_base`, `caml_minor_heaps_end`
2. `minor_heap_area`, `minor_heap_area_end`
3. `young_start`, `young_end`

Given that `young_start` has a long precedent, I would be in favour of the following renaming:

1. `minor_heap_area` -> `minor_heap_area_start`
2. `caml_minor_heaps_base` -> `caml_minor_heaps_start`.

Reply:
Sounds reasonable to me, the naming was inconsistent indeed.